srp/Cargo.toml

```diff
@@ -15,11 +15,14 @@ num-bigint = "0.4"
 generic-array = "0.14"
 digest = "0.9"
 lazy_static = "1.2"
+subtle = "2.4"
 
 [dev-dependencies]
 rand = "0.6"
 sha2 = "0.9"
 sha-1 = "0.9"
+hex = "0.4"
+num-traits = "0.2"
 
 [badges]
 travis-ci = { repository = "RustCrypto/PAKEs" }
```

Comment:
You can get rid of this. We should use GitHub Actions instead.

Reply:
IIRC badges in `Cargo.toml` got deprecated, so we can simply remove it.